Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Registration.hs

```diff
@@ -480,6 +482,7 @@ createPerson req mobileNumber notificationToken mbBundleVersion mbClientVersion
   _ <- Person.create person
   _ <- QDFS.create $ makeIdlePersonFlowStatus person
   _ <- QPS.create createPersonStats
+  fork "update emergency contact id" $ updatePersonIdForEmergencyContacts person.id mobileNumber
```

Comment:
what are we doing here ? trying to link same person as a default emergency contact id for himself ?